frontend/keyboardNav.js

```diff
@@ -26,6 +26,9 @@ var keyCodes = {
   '38': 'up',
   '39': 'right',
   '40': 'down',
+
+  '69': 'expand',
```

Comment:
I added support for using chrome shortcuts by adding it in the manifest file but the shortcut stopped working in the chrome elements panel. Not sure if this is desired.

I also tried this feature in a recently large react app and the expand all on the root node has a small lag and the close all works well. This lag is there even in the elements panel of chrome so this should be acceptable.